packages/gatsby-theme-podcast/src/templates/episode-page.js

```diff
@@ -11,6 +12,51 @@ import SEO from "../components/SEO";
 import { rhythm } from "../utils/typography";
 
 import "./player-styles.css";
+import "./transcript.css";
+
+const TranscriptParagraph = props => {
+  let hasTimestamp = false;
+  const children = React.Children.toArray(props.children).map((child, i) => {
+    if (i === 0 && typeof child === "string") {
+      const timestamp = child.match(/^\[(\d\d):(\d\d)(?::(\d\d))?\]/);
```

Comment:
Check all the `p` tags, only check the first child in each one (since we are checking for `[00:00:00]`

Regex: https://regexr.com/

<img width="356" alt="Screen Shot 2019-08-02 at 1 10 49 AM" src="https://user-images.githubusercontent.com/588473/62345988-686ade00-b4c2-11e9-9828-e7370cc36048.png">

Also did some simple stuff to compute the number of seconds from the timestamp..